### PR TITLE
Add Aqua operators tutorial

### PR DESCRIPTION
**Pull request by jlapeyre**

### CODE REVIEW

**Comment by ikkoham** [comment]
Thanks, can I check this tutorial.

**Comment by jlapeyre** [comment]
@ikkoham Yes, please do check it.

File: tutorials/operators/01_operator_flow.ipynb [hunk 1]
```diff
@@ -0,0 +1,2251 @@
...
+   "execution_count": 53,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "CircuitStateFn(\n",
+      "               \n",
+      "q_0: ───────■──\n",
```
Comment (ikkoham) [inline tutorials/operators/01_operator_flow.ipynb:1675]: Hadamard gate is for the qubit 0. I think `bell = CX @ (I^H) @ Zero` is correct

File: tutorials/operators/01_operator_flow.ipynb [hunk 2]
```diff
@@ -0,0 +1,2251 @@
...
+   "source": [
+    "# Note that XX was the only non-diagonal measurement in our H2 Observable\n",
+    "print(PauliExpectation(group_paulis=False).convert(h2_measurement))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "By default `group_paulis=True`, which will use the `AbelianGrouper` to convert the `SummedOp` into groups of mutually-commuting Paulis. This reduces circuit execution overhead, as each group can share the same circuit execution."
```
Comment (ikkoham) [inline tutorials/operators/01_operator_flow.ipynb:1932]: mutually-commuting Paulis -> mutually qubit-wise commuting Paulis

**Review by ikkoham — COMMENTED** [review]
Very nice note. Let me point out a little.

File: tutorials/operators/01_operator_flow.ipynb [hunk 3]
```diff
@@ -0,0 +1,2251 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Introduction\n",
```
[the remark this answers is not in the document]
Comment (stefan-woerner) [reply tutorials/operators/01_operator_flow.ipynb:7]: just `# Operator Flow`

File: tutorials/operators/01_operator_flow.ipynb [hunk 4]
```diff
@@ -0,0 +1,2251 @@
...
+    "\n",
+    "Aqua provides classes representing states and operators and sums, tensor products, and compositions thereof. These algebraic constructs allow us to build expressions representing operators.\n",
```
Comment (stefan-woerner) [inline tutorials/operators/01_operator_flow.ipynb:9]: best replace `Aqua` by `Qiskit` everywhere.

File: tutorials/operators/01_operator_flow.ipynb (hunk 1 above)
Comment (jlapeyre) [reply tutorials/operators/01_operator_flow.ipynb:1675]: Yes, you are correct. The circuit is clearly wrong.

**Review by ikkoham — APPROVED** [review]
LGTM

**Comment by t-imamichi** [comment]
Qiskit advocates submitted a PR of a tutorial of operators. #1077 
It seems started from Qiskit advocates project. https://github.com/qiskit-community/qiskit-advocate-projects/issues/8

I hope these two groups collaborate.

**Comment by e-eight** [comment]
Looks like we are duplicating efforts. I think we would be happy to collaborate on this.

**Comment by jlapeyre** [comment]
It would really be beneficial to have a tutorial on opflow operators committed sooner rather than later. Can we commit this one and work on integrating #1077 later ?

**Comment by stefan-woerner** [comment]
@jlapeyre I agree, let's merge this version first so that we have a tutorial on the opflow available and then work on further improving it. 

Since it is rather long, can you add a table of content to navigate it? Otherwise I think it is good for now.

**Comment by e-eight** [comment]
@jlapeyre @stefan-woerner Maybe we can make the non-overlapping parts of #1077 in to separate tutorial. Also it will be nice to have a plan for what tutorials would be nice to have, so that the two groups do not repeat the same things.

**Comment by nonhermitian** [comment]
This has two H1 level headers that will break the formatting.

**Comment by jlapeyre** [comment]
> This has two H1 level headers that will break the formatting.

whoops, thought I removed that.

**Review by stefan-woerner — APPROVED** [review]

**Review by nonhermitian — APPROVED** [review]

**Pull request merged**